imap_data_access/file_validation.py

```diff
@@ -538,7 +545,9 @@ def is_valid_date(input_date: str) -> bool:
 
     """Changes needed below here."""
 
-    def construct_path(self) -> Path:
+    def construct_path(
+        self,
+    ) -> Path:  # TODO: how should this path look? Should it be different format?
```

Comment:
Good question - this should be a different function or maybe a flag to differentiate ancillary files and data files. I'm going to ask in slack about where ancillary files should live. 

The initial vote seems to be `imap/ancillary/[instrument]` which also seems like the most logical spot for me. Keep an eye on slack in case that changes, but that's a fine assumption for now.